### ISSUE TITLE
[Feature Request]: Crontab Support

### ISSUE THREAD

**Issue by jgarte** [martanne/vis#1221]
### What feature would you like to see?

I got this error today:

```sh
/usr/share/vis/lexer.lua:1522: no file '/usr/bin/lua/lexers/crontab.lua'
no file '/usr/bin/lua/crontab/init.lua/crontab.lua'
no file '/home/jgart/.config/vis/lexers/crontab.lua'
no file '/home/jgart/.config/vis/crontab/init.lua/crontab.lua'
no file '/etc/vis/lexers/crontab.lua'
no file '/etc/vis/crontab/init.lua/crontab.lua'
no file '/usr/share/vis/lexers/crontab.lua'
no file '/usr/share/vis/crontab/init.lua/crontab.lua'
no file '/usr/share/lua/5.4/lexers/crontab.lua'
no file '/usr/share/lua/5.4/crontab/init.lua/crontab.lua'
no file '/usr/lib64/lua/5.4/lexers/crontab.lua'
no file '/usr/lib64/lua/5.4/crontab/init.lua/crontab.lua'
no file '/crontab.lua'
```

It looks like crontab is not supported.

**Comment by fischerling** [comment]
[scintillua](https://github.com/orbitalquark/scintillua) does not provide a lexer for crontab files.
Maybe you could ask upstream to write a lexer for crontab or submit your own version.

**Comment by mcepl** [comment]
That’s wrong, isn’t it? I thought that supported file type without an appropriate lexer should just default to plain text, not to fail, shouldn’t it?

**Comment by fischerling** [comment]
> That’s wrong, isn’t it? I thought that supported file type without an appropriate lexer should just default to plain text, not to fail, shouldn’t it?

You are totally right, since a7aac1044856abc4d1f133c6563fc604d7fe6295 loading a nonexistent lexer should silently fail.
And I can not reproduce this with my up to date build of vis.

@jgarte What version of vis are you using?

However, this does not change the fact, that there is no lexer for crontab and to get support one should reach out to upstream.

**Comment by jgarte** [comment]
looks like 0.8:

https://toys.whereis.social/?search=vis

@fischerling 

I think I need to update the guix package 🦆 .
